mitodl/open-discussions
# fixes fo ocw-next delete command

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
This shouldn't happen in rc/prod but locally I had two ocw courses with runs with the same slug, which broke the delete command.

Also, I realized that the delete command doesn't actually filter for ocw

This fixes those issues

#### How should this be manually tested?
OCW hugo themes www pointing to your local open discussions database
Run
```
 docker-compose run web ./manage.py backpopulate_ocw_next_data  --course-url-substring anastasia-test-site-21 --overwrite
```
And check that a course named `Anastasia Test Site 21` is added to your local search index

Run
```
docker-compose run web ./manage.py backpopulate_ocw_next_data --delete --course-url-substring anastasia-test-site-21
```
And check that the course has been removed from your local search index
